lib/google_search_data_viewer_web/router.ex

```diff
@@ -40,6 +40,7 @@ defmodule GoogleSearchDataViewerWeb.Router do
     pipe_through [:browser, :authorized]
 
     get "/", KeywordController, :index
+    get "/:id", KeywordController, :show
```

Comment:
How about merging these two routes using the `resources/4` macro with the `:only` option?
```suggestion
    resources "/", KeywordController, only: [:index, :show]
```